public/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title> React Calculator </title>
+  <link rel="stylesheet" href="./styles/styles.css" />
+  <style>
+    html,
+    body {
+      background-image: url('https://www.ecopetit.cat/wpic/mpic/211-2111525_math-background-design.jpg')
+    }
+
+    h1 {
+      text-shadow: 0 0 3px rgb(245, 43, 194), 0 0 5px rgb(245, 43, 194);
+      font-family: "Comic Sans MS", cursive, sans-serif;
+    }
```

Comment:
I appreciate you. Though I'd move this into the css file since you have one, don't split your styles